enterprise/server/remote_execution/container/container.go

```diff
@@ -201,6 +202,14 @@ type CommandContainer interface {
 	Stats(ctx context.Context) (*repb.UsageStats, error)
 }
 
+// Stater can optionally be implemented by a CommandContainer to have
+// its state persisted across executor reboots.
+type Stater interface {
```

Comment:
I would add a `State` method to the container interface and have non-supporting implementations return UNIMPLEMENTED error code, but I'm also okay with this approach if you want to stick with it.

Reply:
Done